Cargo.toml

```diff
@@ -10,7 +10,12 @@ async-trait = "0.1.73"
 clap = "4.4.5"
 derivative = "2.2.0"
 eyre = "0.6.8"
+hex = "0.4.3"
 jsonrpsee = "0.20.1"
-reth = { git = "https://github.com/ultrasoundmoney/reth-block-validator", branch = "changes-to-enable-validation-api-extension" }
+reth = { git = "https://github.com/paradigmxyz/reth" }
```

Comment:
As you can see I just import the latest commit from reth. This could lead to it breaking when they push a breaking change. Should I instead pin a commit ? (Then we miss continous updates though unless we manually update).

Unfortunately there seems to be no releases on crates.io yet so we can't do any more advanced version patterns.

Reply:
Niclas pointed out the lockfile may keep things pinned already.

From [the cargo book](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-git-repositories)

> Once a git dependency has been added, Cargo will lock that dependency to the latest commit at the time. New commits will not be pulled down automatically once the lock is in place. However, they can be pulled down manually with cargo update.

So not as necessary, still think it would be good to be explicit and not have to browse lockfiles to see what code we depend on.

Reply:
Note that I opened an issue in reth asking for more documentation on this:
https://github.com/paradigmxyz/reth/issues/4898